app/views/cases/new.html.slim

```diff
@@ -2,7 +2,7 @@
   = t('.heading')
 
 - content_for :sub_heading
-  = t4c(@case, '', 'sub_heading', case_type: @case.pretty_type)
+  = get_headings(@case, @correspondence_type, '')
```

Comment:
You don't need to pass in the last argument now that you've added a default value to the method.